.vib/discourse/cypress/cypress/integration/discourse_spec.js

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.contains('a', 'Unread');
+  cy.get('#current-user').click();
+  cy.get('[title="Preferences"]').click();
+  cy.contains('Log Out').click();
+  cy.contains('button', 'Log In');
+});
+
+it('allows to sign in', () => {
```

Comment:
It seems like you meant to test the `Sign Up` flow.
To me, Sign in = Log in, which we are testing in the previous test.